src/Faker/Provider/ar_PS/Person.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Faker\Provider\ar_PS;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $maleNameFormats = array(
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{firstNameMale}} {{lastName}}',
+        '{{titleMale}} {{firstNameMale}} {{lastName}}',
+    );
+
+    protected static $femaleNameFormats = array(
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{firstNameFemale}} {{lastName}}',
+        '{{titleFemale}} {{firstNameFemale}} {{lastName}}',
+    );
+
+    /**
+     * @link http://muslim-names.us/
```

Comment:
this site redirects to another one. Reuse license isn't clear. Can you obtain formal agreement from the site rights holders that we can use their data?